Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 RAILS_VERSION = '~> 4.1.8'
 
-ruby ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
+send :ruby, ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
```

Comment:
sadly, I don't think so:
```bash
echo "ruby(ENV['GEMFILE_RUBY_VERSION']) if ENV['GEMFILE_RUBY_VERSION']" | command tr -d '\r' | sed -n "s/[[:space:]]+rescue[[:space:]]+nil$//; /^\s*ruby[[:space:](]/ {s/^\s*ruby//; s/[[:space:]()'\"]//g; p;}"
ENV[GEMFILE_RUBY_VERSION]ifENV[GEMFILE_RUBY_VERSION]
```

It seems that any line beginning with 'ruby' triggers this rvm mechanism